packages/block-editor/src/components/spacing-sizes-control/axial-input-controls.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Internal dependencies
+ */
+import SpacingInputControl from './spacing-input-control';
+import { LABELS } from './utils';
+
+const groupedSides = [ 'vertical', 'horizontal' ];
+
+export default function AxialInputControls( {
+	onChange,
+	values,
+	sides,
+	spacingSizes,
+	type,
+} ) {
+	const createHandleOnChange = ( side ) => ( next ) => {
+		if ( ! onChange ) {
+			return;
+		}
+		const nextValues = { ...values };
+		const nextValue = next;
+
+		if ( side === 'vertical' ) {
+			nextValues.top = nextValue;
+			nextValues.bottom = nextValue;
+		}
+
+		if ( side === 'horizontal' ) {
+			nextValues.left = nextValue;
+			nextValues.right = nextValue;
+		}
```

Comment:
I don't think we need to create `nextValue` here as there is no numerical check like the `BoxControl`.
```suggestion
		if ( side === 'vertical' ) {
			nextValues.top = next;
			nextValues.bottom = next;
		}

		if ( side === 'horizontal' ) {
			nextValues.left = next;
			nextValues.right = next;
		}
```

Reply:
ah, true, will update that.